tests/tools/go/conftest.py

```diff
@@ -0,0 +1,14 @@
+from base64 import urlsafe_b64encode
+from os import urandom
+
+import pytest
+
+from dda.utils.fs import Path
+
+
+@pytest.fixture
+def get_random_filename():
+    def _get_random_filename(k: int = 10, root: Path | None = None) -> Path:
+        return (root or Path()).joinpath(urlsafe_b64encode(urandom(k)).decode("utf-8"))
```

Comment:
This seems to be doing too much. Let's just have a utility that returns a random file name as a `str` and the caller can do what it wants with it.